src/Service.Host/client/src/helpers/utilities.js

```diff
@@ -31,14 +25,9 @@ export const sortList = (list, type) => {
         }
         return 0;
     });
-};
 
-export const sortEntityList = (list, property, type) => {
-    if (type === 'date') {
-        return list.slice().sort((a, b) => moment.utc(a[property]).diff(moment.utc(b[property])));
```

Comment:
Looks like you are removing date sorting?

Reply:
Hmm this file is now labelled Outdated..

Reply:
Yeah @richardiphillips pointed out that as our dates are always iso strings, a string comparison will work.